src/packages/address/existRender.tsx

```
@@ -52,7 +52,7 @@
 
   const onClick: MouseEventHandler = (e) => {
     e.stopPropagation()
-    onSwitch && onSwitch({ type: type === 'exist' ? 'custom' : 'exist' })
+    onSwitch && onSwitch({ type: type === 'exist' ? 'cascader' : 'exist' })
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)